pkg/client/kubernetes/chartoptions.go

```diff
@@ -98,4 +112,7 @@ type DeleteOptions struct {
 	// Forces the namespace for chart objects when applying the chart, this is because sometimes native chart
 	// objects do not come with a Release.Namespace option and leave the namespace field empty
 	ForceNamespace bool
+
+	// Ignores resources with groups, versions or kinds not registered in the apiserver.
+	IgnoreNoMatch bool
```

Comment:
You can rather use 

```suggestion
	TolerateErrorFuncs []client.TolerateErrorFunc
```

Ref https://github.com/gardener/gardener/blob/a6f8547e7926b767f15f525bc45ace39a81ea140/pkg/utils/kubernetes/client/options.go#L26-L27